src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -1549,6 +1549,8 @@ void GrpcLb::PendingPickSetMetadataAndContext(PendingPick* pp) {
 void GrpcLb::OnPendingPickComplete(void* arg, grpc_error* error) {
   PendingPick* pp = static_cast<PendingPick*>(arg);
   PendingPickSetMetadataAndContext(pp);
+  // We can run the original closure safely here because it is run in the same
```

Comment:
Instead of "we can run the original closure", please say "we can use GRPC_CLOSURE_RUN() instead of GRPC_CLOSURE_SCHED()", to make it clear to the reader what this is referring to.

Reply:
Done.

Squashing.